lib/kite/core.rb

```diff
@@ -47,13 +47,16 @@ def generate()
 
     desc 'render MANIFEST', 'Render manifest file from configuration and Terraform output'
     def render(manifest)
+      (say ".tfstate file is not present, did you terraform apply?", :red; return false) unless File.file? "terraform/terraform.tfstate"
```

Comment:
Please make a class / helper
We can do a series of error helpers for testing missing configs